apps/examples/board_specific_demo/ameba_secure_world_demo/secure_world.c

```diff
@@ -0,0 +1,78 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+
+#define SHOW_USAGE printf("Usage: secure_world [opt2] secure_world_address [opt1]\n \
+		opt1(-r/-w): One of the option is mandatory\n \
+		-r: run the read test\n \
+		-w: run the write test\n \
+		opt2(-a):\n \
+	        -a: secure_world_address: take the address as input, default: 0x735C021\n \
+		Valid Range: 0x7034C020 - 0x7039C000\n \");
+
+#ifdef  CONFIG_AMEBALITE_TRUSTZONE
+volatile uint32_t  *address = 0x735C021;
+#endif
```

Comment:
How about adding it in Kconfig because it's a different value for every board?
If we add it as Kconfig, we can keep the same code for every board. (No need to modify the code itself)